spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+describe UsersController, :type => :controller do
+  it 'can create user' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.